test-harness/terratest-extensions/modules/azure/datafactory.go

```diff
@@ -0,0 +1,253 @@
+package azure
```

Comment:
Perhaps from a different PR? Can you remove this, unless it needs to be here for some reason?